test/unit/integrations/realex/realex_module_test.rb

```diff
@@ -4,7 +4,7 @@ class RealexModuleTest < Test::Unit::TestCase
   include OffsitePayments::Integrations
 
   def test_helper_method
-    assert_instance_of Realex::Helper, Realex.helper(123, 'test')
+    assert_instance_of Realex::Helper, Realex.helper('123', 'test')
```

Comment:
You need to pass in a string or joining the `order + @timestamp` throws an exception